frontend/src/pages/ActivityReport/Pages/components/ObjectiveSelect.js

```diff
@@ -43,6 +62,37 @@ export default function ObjectiveSelect({
           components={components}
         />
       </Label>
+
+      <Modal
```

Comment:
Should we be using our custom modal component here.

Reply:
ah yes 6de96fc